webpackShims/moment-timezone.js

```diff
@@ -0,0 +1,2 @@
+var moment = module.exports = require('../node_modules/moment-timezone/moment-timezone');
+moment.tz.load(require('../node_modules/moment-timezone/data/packed/latest.json'));
```

Comment:
Maybe add `json` to https://github.com/jbudz/kibana/blob/issues/1600/src/optimize/BaseOptimizer.js#L133 and require `latest` instead of `latest.json`? Then we would be more in line with the way `require` work in node. (On that note, I'm not a big fan of `less` being there. Keeping only JS stuff there seems like the natural choice — but that's a discussion way outside of this PR)